src/scenes/RecipeDetails.js

```diff
@@ -34,9 +35,15 @@ const recipe = {
 //   sevings={recipe.sevings}
 // />
 
-export default function RecipeDetails({ image, title, description, nutrients, kcal, sevings }) {
+export default function RecipeDetails(props) {
     
     const styles = useStyles();
```

Comment:
This is from older commit, but try not to use "px" values in IconLeft and IconRight as it can be displayed incorrectly on smaller or bigger screens. What is more, they should be in my opinion a little bit more "to the outside", not "to the center".